src/modules/mount/main.py

```diff
@@ -307,6 +307,12 @@ def mount_partition(root_mount_point, partition, partitions, mount_options, moun
                                         mount_option) != 0:
                 libcalamares.utils.warning("Cannot mount {}".format(device))
 
+def enable_swap_partition(swap_device):
+    try:
+        for d in swap_device:
+        	subprocess.check_call(["swapon", d])
+    except subprocess.CalledProcessError:
```

Comment:
It looks like we reverted back to subprocess at some point in the many merges.  This should use `host_env_process_output()` instead.

Reply:
OK